apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/BBCodingSystems.java

```diff
@@ -16,7 +16,7 @@
   public static final String CLM_POA_IND = BB_BASE_URL + "/variables/clm_poa_ind_sw1";
 
   public static final String FI_DOC_CLM_CONTROL_NUM =
-      BB_BASE_URL + "/variables/fi_doc_clm_cntrl_num";
+      BB_BASE_URL + "/variables/fi_doc_clm_cntl_num";
```

Comment:
@KyRobbins do we need to change these URLs to use the `/codesystem/` path?

Reply:
This path has been abandoned